packages/repository/src/repository-mixin.ts

```diff
@@ -113,5 +113,16 @@ export function RepositoryMixin<T extends Class<any>>(superClass: T) {
         }
       }
     }
+
+    /**
+     * If @loopback/boot is enabled, find and mount all repositories
+     * automatically at boot time.
```

Comment:
I am worried about packaging bootstrapper with repository runtime. If we decide to make a breaking change in our boostrapper, we will have to release a new semver-major version of the repository package too (which may be more expensive/difficult to do). It will also force users of the `repository` to always use the latest `boot` version (and project conventions), therefore making the cost of upgrading to a newer semver-major version of the `repository` package more expensive.

Is there a way how to move bootstrapping of repositories to the `boot` package and export it as an optional step, where either the user can opt-in/opt-out, or else the bootstrapper detects whether the target `app` has `repository` method and use that to decide whether to boot repositories or not?

Ideally, this boot step would belong to its own package, e.g. `@loopback/repository-boot`, similarly to how https://github.com/strongloop/loopback-next/pull/740 is introducing `@loopback/repository-rest`, but I am not sure if that's not an overkill, to have such a small package?